### ISSUE TITLE
Get a 403/404 error when preform login, create new task on web app

### ISSUE THREAD

**Issue by ing-oasis** [opencv/cvat#4981]
Hi.

I have repeated errors (response from the console is 403 or 404) when login webpage (app.cvat.ai) or after login in successfully and creating a new task on my project. It sometimes gives Network error.

My projectId=1773 and the error from the webpage is:

HTTP 403 Forbidden
Allow: GET, POST, DELETE, HEAD, OPTIONS
Content-Type: application/vnd.cvat+json
Vary: Accept

{
    "detail": "You do not have permission to perform this action."
}

Weirdly, when I try to use the local server in docker, I can see my previous project, but it gives a network error when creating a new task.

Contact me if you need any information, and thanks for your amazing work.

**Comment by nmanovic** [comment]
@ing-oasis ,

How many tasks you already have on CVAT.ai?

**Comment by ing-oasis** [comment]
Hi, Nikita.

There are 15 tasks in one project, including 4 validation tasks and 11
train tasks.

I can create a new project, but can not add my task to it. The data I want
to upload is a file containing 14000 images, and that would raise network
error.

I tried to upload all images and failed again. After moving 40 images into
one file manually, I dragged the file to the upload area and failed again.

The last time I clicked the upload area, selected all 40 images, then it
succeeded.

Can not figure it why, but I can start my work now.

Hope it helps.

Thanks for your hard work.

Regards
Ding
